Cargo.toml

```
@@ -53,3 +54,7 @@ glob = "0.3.1"
 
 [profile.dev.package."*"]
 opt-level = 3
+
+[[bench]]
+name ="benchmark"
```

style nit (not sure why `cargo fmt` doesn't format the Cargo toml files)
```suggestion
name = "benchmark"
```